resources/web/wwi/resource_manager.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+function ResourceManager() {
```

Comment:
What does this class do? A comment would be welcome. Also the name `ResourceManager` is not very useful. What kind of resources are we talking about? A better name could be `wwi_url.js`.

Reply:
The idea was to have a class that handles all the references to (local) resources like images, etc. without having to hide these paths some other code.

I would prefer a more generic name than `wwi_url.js`.

Reply:
Maybe we could call this class `StandardPath` as in Webots

Reply:
To me, it looks more related with URLs rather than paths, isn't it? What about `DefaultUrl`?